bitnami/argo-cd/values.yaml

```
@@ -554,6 +554,397 @@ controller:
   ##
   initContainers: []
 
+## @section Argo CD ApplicationSet controller parameters
+
+## ApplicationSet controller
+##
+applicationSet:
+  ## @param applicationSet.enabled Enable ApplicationSet controller
+  enabled: false
```

As an standard we are adding a new line with `##` between the param description and the parameter itself
```suggestion
  ## @param applicationSet.enabled Enable ApplicationSet controller
  ##
  enabled: false
```

Could you check other cases as well?